src/term.rs

```diff
@@ -625,6 +631,7 @@ mod test {
         qmark, QMark, "?";
         or, Or, "or";
         optional, Optional, "optional";
+        r#async, Async, "async";
```

Comment:
This looks a bit weird in the test output...
```
test term::test::r#async::should_parse ... ok
test term::test::r#async::should_parse_if_anything_next ... ok
test term::test::r#async::should_parse_with_preceding_spaces ... ok
test term::test::r#async::should_parse_with_succeeding_spaces ... ok
test term::test::r#async::should_parse_with_surrounding_spaces ... ok
```
Suggestions welcome.

Reply:
...I really should just look around a bit more first. Terms like [`enum`](https://github.com/rustwasm/weedle/blob/c7374a5ae0f3af0a659cfbdd7305b78b1404f4e8/src/term.rs#L633) or [`static`](https://github.com/rustwasm/weedle/blob/c7374a5ae0f3af0a659cfbdd7305b78b1404f4e8/src/term.rs#L645) have the same problem, and they just put an underscore at the end. 😅